api/openapi.yaml

```diff
@@ -167,6 +176,27 @@ paths:
           $ref: "#/components/responses/BadRequestProblemResponse"
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
+  /api/v1alpha2/namespaces:
+    post:
+      description: Create namespace
+      operationId: createNamespace
+      tags:
+        - Meters
```

Comment:
Should this belong to ingest or it's own category?

Reply:
Both Ingest and Meters should be added.